### DISCUSSION TITLE
UHD 630 GPU Problem

### DISCUSSION THREAD

**Discussion by ferdisahin** [ansonliao/Asrock-B460m-ITX-AC-OC-EFI#40] [Q&A]
UHD 630 graphics card works but is very slow in Figma's desktop app. There is lag while working. Same problem happens when scrolling page in VSCode. Do you also have such a problem?

**Comment by ansonliao** [comment]
@ferdisahin 
Let me try Figma.
BTW you can try this app to improve the smooth of the mouse scrolling: https://github.com/Caldis/Mos
